com.unity.netcode.gameobjects/Runtime/SceneManagement/NetworkSceneManager.cs

```diff
@@ -1496,7 +1496,18 @@ private void OnClientBeginSync(uint sceneEventId)
                 // don't try to reload it.
                 shouldPassThrough = true;
             }
-
+            else
```

Comment:
This will cause conflicts with multiple versions of the same additive scene being loaded since their names will be identical and the only distinguishing difference being the handle of the scene loaded.
However, I think I could make a change to the ValidateSceneBeforeLoading to bypass loading if you return false without completely exiting the synchronization process which would give you the same functionality I believe you are looking for?

Reply:
Hadn't encountered a situation where one has to load the same scene several times but your solution sounds much better - more control to the API user, yeah ;-)